R/utils-check.R

```diff
@@ -164,3 +164,21 @@ arg_match_compat_level <- function(arg, arg_nm = caller_arg(arg), error_call = c
     )
   }
 }
+
+# Same as Python Polars' `_to_sink_target`
+arg_to_sink_target <- function(arg, arg_nm = caller_arg(arg), error_call = caller_env()) {
+  if (is_string(arg)) {
+    arg
+  } else if (inherits(arg, "polars_partitioning_scheme")) {
```

Comment:
Should we have `is_polars_partitioning_scheme` for consistency with the other classes?